### ISSUE TITLE
mrview: -voxel -target don't work

### ISSUE THREAD

**Issue by maxpietsch** [MRtrix3/mrtrix3#1685]
In dev (`3.0_RC3-912-gf93fbf2f`) but also on master, `-voxel` and `-target` options are processed but have not effect.

Example: `mrview w1.mif -voxel 0,0,0` opens in the centre of the image, not at `0,0,0`.

 It is likely a race condition as it sometimes works with
`mrview w1.mif -load w1.mif -mode 2 -voxel 0,0,0`.

Qt version 5.5.1, ubuntu 16.04

**Comment by Lestropie** [comment]
This might not be as specific an issue as reported; pretty sure there's a lot of race conditions in `mrview`, perhaps even more so when it comes to command-line option parsing, which is part of the basis for #1437. E.g. See one of the screenshots [here](https://github.com/MRtrix3/mrtrix3/pull/1228#issuecomment-381514370), where for one image at least one command-line option wasn't parsed correctly, but the rest were, including `-screenshot.grab`.

**Comment by jdtournier** [comment]
Yes, command-line parsing is messy because of the way a lot of the work is done during screen updates, so the screen needs to be redrawn after each option, and that's a bit tricky to achieve properly with Qt... I'd like to find the time to deal with #1437 properly, but I can't see that happening any time soon. 😒
